paasta_tools/cli/cmds/logs.py

```diff
@@ -328,16 +372,37 @@ def get_log_reader():
 
 
 class LogReader(object):
-    def __init__(self, **kwargs):
-        pass
+    def __init__(self):
+        # Tailing, i.e actively viewing logs as they come in
+        self.SUPPORTS_TAILING = False
```

Comment:
aah okay, that makes sense. LogReader is a singleton right now which is why I had it ass instance level attributes. Fixing.